src/saltext/vmware/states/esxi.py

```diff
@@ -905,56 +905,68 @@ def advanced_config(
     service_instance
         Use this vCenter service connection instance instead of creating a new one. (optional).
 
+    profile
+        Profile to use (optional)
+
+    less
+        Default False. If this is set to True, only the changed values will be reported as changes.
+
     .. code-block:: yaml
 
-        Remove User:
+        ESXi_advanced_config_example:
           vmware_esxi.advanced_configs:
-            - name: Annotations.WelcomeMessage
-            - value: Hello
-
+            - less: True
+            - configs:
+              key_1: value_1
+              key_2: value_2
```

Comment:
**suggestion** these should be indented 2 more spaces, otherwise it's invalid Jinja

```suggestion
                key_1: value_1
                key_2: value_2
```